### DISCUSSION TITLE
How could we change/custom the logo?

### DISCUSSION THREAD

**Discussion by latiosthinh** [firecmsco/firecms#383] [Q&A] [answered]
Is there any way to custom logo of the app?

![image](https://user-images.githubusercontent.com/7105120/190578936-95c4258d-8cca-4238-8e2e-1a24b7b71e07.png)

**Comment by fgatti675** [comment, marked as answer]
Hi @latiosthinh 
Sure, you can simply pass your imported image in the `logo` prop of `FirebaseCMSApp`

**Reply by latiosthinh** [reply]
Took me a while but this kind of customization should be written in Docs - I think
Thank you so much.

**Comment by botzyus** [comment]
I was able to change the logo using
return <Scaffold
                            autoOpenDrawer={false}
                            logo="/logo.png">
                            <AppBar title={"FORM Space CMS"}/>
                            <Drawer/>
                            <NavigationRoutes/>
                            <SideDialogs/>
                        </Scaffold>;
  - added line    :
   logo="/logo.png"
   - logo.png is in public folder
